packages/app-accounts/src/modals/Create.tsx

```diff
@@ -173,13 +175,14 @@ function Create ({ className, onClose, onStatusChange, seed: propsSeed, t, type:
   };
   const _onChangeName = (name: string): void => setName({ isNameValid: !!name.trim(), name });
   const _toggleConfirmation = (): void => setIsConfirmationOpen(!isConfirmationOpen);
+  const context = useContext<MyAccountContextProps>(MyAccountContext);
```

Comment:
There is a ready to use function exactly for this: `useMyAccount()`
```suggestion
import { useMyAccount } from '@polkadot/joy-utils/MyAccountContext';
// ...
  const context = useMyAccount();
```